root/static/scripts/homepage/language.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * @flow strict
+ * Copyright (C) 2025 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+
+import * as React from 'react';
+
+import {SanitizedCatalystContext} from '../../../context.mjs';
+import {returnToCurrentPage} from '../../../utility/returnUri.js';
+import languageIcon from '../../images/homepage/language-icon.svg';
+import {l} from '../common/i18n.js';
+import {capitalize} from '../common/utility/strings.js';
+
+function languageName(
+  language: ?ServerLanguageT,
+  selected: boolean,
+) {
+  if (!language) {
+    return '';
+  }
+
+  const {
+    id,
+    native_language: nativeLanguage,
+    native_territory: nativeTerritory,
+  } = language;
+
+  let text = `[${id}]`;
+
+  if (nativeLanguage) {
+    text = capitalize(nativeLanguage);
+
+    if (nativeTerritory) {
+      text += ' (' + capitalize(nativeTerritory) + ')';
+    }
+  }
+
+  if (selected) {
+    text += ' \u25be';
+  }
```

Comment:
This bit should probably be dropped now. It means the active item gets a ▾ after it which made sense when that language would be the menu header, but not now.